src/extension/background-script/actions/setup/validateAccount.js

```diff
@@ -7,9 +7,14 @@ const validateAccount = async (message, sender) => {
 
   try {
     const info = await connector.getInfo();
-    await connector.unload(); // unload the connector again, we just checked if it works but have no persistence
```

Comment:
Remove this? Doesn't that change the behavior for adding accounts in the extension?